manifest.md

```diff
@@ -28,15 +28,15 @@ De rest van de standpunten in dit manifest werken verschillende aspecten van dez
 
 ## 2. Patiënt centraal
 
-Wij geloven dat de patiënt een actieve plaats verdient in het netwerk, bijvoorbeeld via haar PGO. De patiënt of een gemachtigde mantelzorger of bewindvoerder moet ervoor kunnen kiezen om mee te praten of toestemmingen te beheren en kunnen zien welke informatie er wordt uitgewisseld.
+Wij geloven dat de patiënt een actieve plaats verdient in het netwerk, bijvoorbeeld via haar PGO (persoonlijke gezondheidsomgeving). De patiënt of een gemachtigde mantelzorger of bewindvoerder moet ervoor kunnen kiezen om mee te praten of toestemmingen te beheren en kunnen zien welke informatie er wordt uitgewisseld.
 
 <p><details markdown="1"><summary>Lees verder...</summary>
 
 Van het doorgeven van de meterstanden tot een weekendje uit boeken: technologie stelt mensen in staat om meer en meer zaken zelf te regelen en in de gaten te houden. Maar in de zorg leveren we onszelf over aan de professional en moeten we vervolgens hopen dat onze belangen behartigd worden.
 
-De verhouding tussen arts en patient is al jaren aan het veranderen. Met slimme E-health oplossingen, zelfmetingen en PGO's zal de patiënt een actievere rol kunnen innemen in zijn eigen gezondheid. Het zwaartepunt van zorg zal zich gaan verplaatsen van behandelen naar preventie. Daarom hebben we softwareapplicaties nodig die de patiënt in staat stellen om mee te praten, toestemmingen te beheren en te zien welke informatie er wordt uitgewisseld tussen de betrokken partijen.
+Maar die verhouding tussen arts en patiënt is de laatste jaren aan het verschuiven. Dankzij slimme _E-health_ oplossingen, zelfmetingen en PGO's zullen we een actievere rol kunnen spelen in onze eigen gezondheid. Het zwaartepunt zal zich gaan verplaatsen van genezen naar voorkomen. Maar daarvoor hebben we softwareapplicaties nodig die ons in staat stellen om mee te praten, toestemmingen te beheren en te zien welke informatie er over ons wordt uitgewisseld.
```

Comment:
Kunnen we alineas niet beginnen met een verbindingswoord. Dus deze zin omschrijven naar: 
>De verhouding tussen arts en patient...

Reply:
Nevermind, ik vind hem wel gepast 👍